### PR TITLE
*: Support set variable to default value.

### PR DESCRIPTION
**Pull request by shenli**
To set a SESSION variable to the GLOBAL value or a GLOBAL value to the compiled-in MySQL default value, use the DEFAULT keyword.
See http://dev.mysql.com/doc/refman/5.7/en/set-statement.html
@coocood @zimulala @tiancaiamao

### CODE REVIEW

File: executor/executor_simple_test.go [hunk 1]
```diff
@@ -110,6 +110,21 @@ func (s *testSuite) TestSetVar(c *C) {
...
+	// For global var
+	tk.MustQuery(`select @@global.low_priority_updates;`).Check(testkit.Rows("OFF"))
+	tk.MustExec(`set @@global.low_priority_updates="ON";`)
+	tk.MustQuery(`select @@global.low_priority_updates;`).Check(testkit.Rows("ON"))
+	tk.MustExec(`set @@global.low_priority_updates=DEFAULT;`) // It will be set to compiled-in default value.
+	tk.MustQuery(`select @@global.low_priority_updates;`).Check(testkit.Rows("OFF"))
+	// For session
+	tk.MustQuery(`select @@session.low_priority_updates;`).Check(testkit.Rows("OFF"))
+	tk.MustExec(`set @@global.low_priority_updates="ON";`)
+	tk.MustExec(`set @@session.low_priority_updates=DEFAULT;`) // It will be set to global var value.
```
Comment (zimulala) [inline executor/executor_simple_test.go]: `low_priority_updates` will be global var value.
Does it means the result of `select @@global.low_priority_updates` is `on` when we remove line 124.

Comment (zimulala) [inline executor/executor_simple_test.go]: Ignore it.

**Comment by zimulala** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Pull request merged**